common/src/main/java/org/opensearch/ml/common/transport/register/MLRegisterModelInput.java

```diff
@@ -72,6 +72,7 @@ public class MLRegisterModelInput implements ToXContentObject, Writeable {
     private List<String> backendRoles;
     private Boolean addAllBackendRoles;
     private AccessMode accessMode;
+    private Boolean isThisVersionCreatingModelGroup;
```

Comment:
What's this for?

Reply:
We use this later to delete the model group if version registration fails

Reply:
Can user set this parameter?

Reply:
It is optional. This is mainly for us to use later in the code. Even if user sets it wrong, we will set it right based on the other input parameters provided